eclair-core/src/main/scala/fr/acinq/eclair/channel/Helpers.scala

```diff
@@ -565,6 +565,13 @@ object Helpers {
       }
     }
 
+    /** Compute the fee paid by a commitment transaction. */
+    def commitTxFee(commitInput: InputInfo, commitTx: Transaction, isFunder: Boolean): Satoshi = {
```

Comment:
So this small method allowed you to remove the huge `def networkFeePaid()` one? Where is the trade-off? Is it only that we store published *and* confirmed transactions, instead of just confirmed? Seems too good to be true 😅 . And on top of that we are more precise IIUC. And we remove the duplicate tx desc hardcoded in `networkFeePaid`. Gosh.

Reply:
Yes, the trade-off is that we're storing published transactions that may never confirm, but I think it's actually a good auditing feature so I believe it's a net win!